package.json

```diff
@@ -8,8 +8,9 @@
   ],
   "scripts": {
     "build": "node build.js",
-    "transform-base": "npx token-transformer data data/transformed-core.json core --expandTypography",
-    "transform": "yarn transform-base"
+    "transform-core": "npx token-transformer data data/transformed-core.json core --expandTypography",
+    "transform-light": "npx token-transformer data data/transformed-light.json core,light core",
+    "transform": "yarn transform-core && yarn transform-light"
```

Comment:
do we hate this combo pack script?

Reply:
Nah, but I like `transform-core`! Nice!

Reply:
Why do we have `core,light`, and then `core`? Is that not just excluding the `core` that we already included?

Reply:
AHH, okay lol, for some reason I was like "Why do you need to include core", but then I realized its cause `light` has references to core, and thus you need both to transform them

Reply:
exactly!